### PR TITLE
fix another large batch of tests of FreeBSD

### PR DESCRIPTION
**Pull request by mkrzewic**

### CODE REVIEW

**Comment by pcanal** [comment]
@phsft-bot build

File: root/meta/genreflex/ROOT-5768/RelationalCool/scripts/coolSqlplus.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 # See https://twiki.cern.ch/twiki/bin/view/PSSGroup/RlWrap
 if [ "${SCRAM_ARCH:0:5}" = "win32" ]; then
```
Comment (pcanal) [inline root/meta/genreflex/ROOT-5768/RelationalCool/scripts/coolSqlplus.sh:3]: I am guessing that this needs to be checked ...

Comment (mkrzewic) [reply root/meta/genreflex/ROOT-5768/RelationalCool/scripts/coolSqlplus.sh:3]: @pcanal yes, I missed that one, pushing fix...

**Review by pcanal — APPROVED** [review]

**Comment by pcanal** [comment]
@phsft-bot build

**Comment by mkrzewic** [comment]
updated a couple of things sonatype found.

**Review by hahnjo — APPROVED** [review]
Also LGTM. Can we merge this @pcanal ?

**Pull request merged**